File: 1. Begginer/10- Pida al usuario dos números reales (que se guardarán en las variables dato1 y dato2).Si dato2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resultad/main.py
data1 = float(input('Ingrese un número real: '))
data2 = float(input('Ingrese otro número real: '))
if data2 == 0 :
	print('Error: No se puede dividir un número entre 0')
else :
	print('El resultado de dividir ' + str(data1) + ' entre ' + str(data2) + ' es: ' + str(data1/data2))

File: 1. Begginer/10- Pida al usuario dos números reales (que se guardarán en las variables dato1 y dato2).Si dato2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resultad/main.php
<?php 
/* Float en php sería de 14 dígitos */
(float)$data1 = readline('Ingrese un número real: ');
(float)$data2 = readline('Ingrese otro número real: ');
if ($data2 == 0) {
	echo 'Error: No se puede dividir entre 0';
} else {
	echo 'El resultado de dividir ' . $data1 . ' entre ' . $data2 . ' es: ' . (float)$data1/$data2;
}